services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OrcidOAuth2Provider.java

```diff
@@ -87,12 +69,30 @@ public static String parseOrcidId(String body){
 		}
 	}
 
-
+	public static AliasAndType parseAliasAndType(Token accessToken) {
```

Comment:
rename accessToken to accessTokenResponse
maybe add a comment:
// https://github.com/ORCID/ORCID-Source/blob/main/orcid-web/ORCID_AUTH_WITH_OPENID_CONNECT.md